decentraland/unity-renderer
# feat: friend entries loaded by the scroll position & visual improvements on the loading spinners

## What does this PR change?

- friend entries loaded when you scroll down to the bottom on friend list and friend request list
- the "show more" option has been changed adding a loading spinner & a label modification
- the loading spinner style has been changed

## How to test the changes?

1. Open the friends window from the taskbar
2. While the friend service is not initialized, check that the loading spinner has been resized to 40px
3. Force the initialization of friends
4. Scroll down to the bottom of the friend list
5. There should be an ui component indicating that we are loading more entries
6. Load all the available entries, the "show more" component should be hidden
7. Go to friend request tab and repeat the process

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
